crates/turbopack-ecmascript/src/analyzer/linker.rs

```diff
@@ -16,6 +16,8 @@ type LinkCacheValue = (
     Vec<Vec<(HashSet<Id>, Vec<(HashSet<Id>, JsValue)>)>>,
 );
 
+type LinkCacheItem = Option<(bool, JsValue, (HashSet<Id>, HashSet<Id>))>;
```

Comment:
It's worth making this a struct as well.

Reply:
Done